### PR TITLE
Add Feature List to Readme Chnage Name and Description

### PR DESCRIPTION
**Pull request by YisroelTech**
Perhaps a bit to געוואגט... I changed the extension name and description to something reflecting better the nature of the extension.

@mordechairoth Let me know if this is a No No.

I also added a short feature list in the README.

(I think you can add it to the Chrome Store listing, and also add the GitHub page as the Website for the extension.)

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "update_url": "https://clients2.google.com/service/update2/crx",
     "manifest_version": 2,
-    "name": "ivelt keyboard shortcuts",
-    "description": "enabe to go forward or backward in ivelt forums by pressing the right and left buttons, and other accessibility features for ivelt",
+    "name": "iVelt Forum Helper",
+    "description": "Add Acessiblilty features, and other usability oimprovemtns to the iVelt Forums.",
```
Comment (mayim-chayim) [inline src/manifest.json:5]: typo oimprovemnts

File: README.md [hunk 2]
```diff
@@ -3,3 +3,9 @@ Browser Extension to add accessibility features to www.ivelt.com
 
 To download the extension please go to the Chrome Web Store:
 https://chrome.google.com/webstore/detail/ivelt-keyboard-shortcuts/bcfoanjpmoikpanagkinedlihjbaijke
+
+## Extension features:
+* Go forward or backward in ivelt forums by pressing the right and left keyboard buttons
+* Submit post by presseing <kbd>Ctrl</kbd>+<kbd>Enter</kbd>
+* See when you receive inbox notifications on iVelt, it adds an icon tothe toolbar and updates the notification badge number when you have new notification
```
Comment (mayim-chayim) [inline README.md:10]: typo tothe

File: src/manifest.json (hunk 1 above)
Comment (YisroelTech) [reply src/manifest.json]: Corrected

File: README.md (hunk 2 above)
Comment (YisroelTech) [reply README.md]: Corrected

**Comment by mordechairoth** [comment]
I actually like the idea of updating the name and desctiption! hope to review this tomorrow or later this week as it is not urgent.

**Review by bevekashaivelt — COMMENTED** [review]
Typo - Submit post by presseing, should be pressing

File: src/manifest.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "update_url": "https://clients2.google.com/service/update2/crx",
     "manifest_version": 2,
-    "name": "ivelt keyboard shortcuts",
-    "description": "enabe to go forward or backward in ivelt forums by pressing the right and left buttons, and other accessibility features for ivelt",
+    "name": "iVelt Forum Helper",
+    "description": "Add Acessiblilty features, and other usability improvements to the iVelt Forums.",
```
Comment (Sunegi) [inline src/manifest.json:5]:
```suggestion
    "description": "Adds accessibility features, and other usability improvements to the אייוועלט forums.",
```

**Pull request merged**